### PR TITLE
2019.1.8

### PR DESCRIPTION
**Pull request by FajsiEx**
another version i guess. _**Thanks to Cody.**_

### CODE REVIEW

**Comment by FajsiEx** [comment]
!mtc bug resolved.

File: modules/commands/lyrics/mtc.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+And she would sing
+And she would sing
+And she would sing
+(and she would sing) (and she would sing)
+(and she would sing) (and she would sing)
+                `
+            }
+        });
+    }
+}
```
[the remark this answers is not in the document]
Comment (FajsiEx) [reply modules/commands/lyrics/mtc.js]: Yep ill fix this thx

File: modules/commands/lyrics/mtc.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
+const COLORS = require("../../consts").COLORS;
```
[the remark this answers is not in the document]
Comment (FajsiEx) [reply modules/commands/lyrics/mtc.js]: https://stackoverflow.com/questions/27441803/why-does-jshint-throw-a-warning-if-i-am-using-const easy enough

File: modules/commands/jff/kubko.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+                    "embed": {
+                        "title": "Error",
+                        "color": COLORS.RED,
+                        "description": "Vyskytla sa chyba pri requestovaní random postu z redditu."
+                    }
+                });
+            }
+        });
+    }
+}
```
[the remark this answers is not in the document]
Comment (FajsiEx) [reply modules/commands/jff/kubko.js]: Fix this also

**Comment by FajsiEx** [comment]
Should fix all the hound issues...

**Comment by CodyM4V3R1CK** [comment]
welp, this should be ready for merge

**Comment by FajsiEx** [comment]
merge denied.
review needed.

**Comment by CodyM4V3R1CK** [comment]
!oof

**Comment by FajsiEx** [comment]
ready for merge @CodyM4V3R1CK ill let you do the honors

**Pull request merged**